cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5377,6 +5377,15 @@
     </description>
   </property>
 
+  <property>
+    <name>feature.streaming.pipeline.native.state.tracking.enabled</name>
+    <value>false</value>
+    <final>false</final>
```

Comment:
don't need this if it is false. Or is this here so we can switch it to true in certain Docker builds?

Reply:
Removed this , will add if required later.